projects/knora-ui/src/lib/viewer/values/interval-value/interval-value.component.html

```diff
@@ -1,22 +1,29 @@
-<span [formGroup]="form">
-    <mat-form-field class="large-field child-value-component" floatLabel="never">
-      <kui-interval-input #intervalInput [formControlName]="'intervalValue'" class="value" placeholder="Interval value" [readonly]="mode === 'read'" [errorStateMatcher]="matcher"></kui-interval-input>
-      <mat-error *ngIf="form.controls.intervalValue.hasError('valueNotChanged')">
-        Change value
-      </mat-error>
-    </mat-form-field>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-      <textarea matInput
-                cdkTextareaAutosize
-                #autosize="cdkTextareaAutosize"
-                cdkAutosizeMinRows="1"
-                cdkAutosizeMaxRows="6"
-                [formControlName]="'comment'"
-                class="comment"
-                placeholder="Comment"
-                type="text"
-                [readonly]="mode === 'read'"
-                spellcheck="false">
-        </textarea>
-    </mat-form-field>
-</span>
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+  <p class="rm-value interval-start">Start: {{valueFormControl.value.start}}</p>
+  <p class="rm-value interval-end">End: {{valueFormControl.value.end}}</p>
+  <p class="rm-comment" *ngIf="shouldShowComment">{{commentFormControl.value}}</p>
+</div>
+<ng-template #showForm>
+  <span [formGroup]="form">
+      <mat-form-field class="large-field child-value-component" floatLabel="never">
+        <kui-interval-input #intervalInput [formControlName]="'intervalValue'" class="value" placeholder="Interval value" [readonly]="mode === 'read'" [errorStateMatcher]="matcher"></kui-interval-input>
```

Comment:
see comment above about readonly.

Reply:
Will be done in https://dasch.myjetbrains.com/youtrack/issue/DSP-206